File: Beecrowd/Beginner/1000 - 1099/1012 - Area/Python/1012_Area.py
a, b, c = list(map(float, input().split()))

triangle = (a * c) / 2
circle = 3.14159 * c * c
trapezium = ((a + b) * c) / 2
square = b * b
rectangle = a * b

print(f"TRIANGULO: {triangle:.3f}")
print(f"CIRCULO: {circle:.3f}")
print(f"TRAPEZIO: {trapezium:.3f}")
print(f"QUADRADO: {square:.3f}")
print(f"RETANGULO: {rectangle:.3f}")

File: Beecrowd/Beginner/1000 - 1099/1012 - Area/C/1012_Area.c
#include <stdio.h>

int main() {
    double a, b, c;
    scanf("%lf %lf %lf", &a, &b, &c);

    double triangle = (a * c) / 2;
    double circle = 3.14159 * c * c;
    double trapezium = ((a + b) * c) / 2;
    double square = b * b;
    double rectangle = a * b;

    printf("TRIANGULO: %.3lf\n", triangle);
    printf("CIRCULO: %.3lf\n", circle);
    printf("TRAPEZIO: %.3lf\n", trapezium);
    printf("QUADRADO: %.3lf\n", square);
    printf("RETANGULO: %.3lf\n", rectangle);
    
    return 0;
}
